packages/babel-helper-fixtures/src/index.js

```diff
@@ -1,6 +1,6 @@
-import cloneDeep from "lodash/cloneDeep";
-import clone from "lodash/clone";
-import extend from "lodash/extend";
+import cloneDeep from "lodash.clonedeep";
+import clone from "lodash.clone";
```

Comment:
We could avoid `clone`:

```js
      options: { ...rootOpts },
```

Reply:
Thanks for the confirmation @nicolo-ribaudo - this is now removed in https://github.com/babel/babel/pull/11811, along with one other `clone` callsite